app/common/tables.html

```diff
@@ -0,0 +1,22 @@
+<div class="col-sm-8">
+   <center>
+      <h4>View Poll Data</h4>
+      {{OutletCtrl.outletdata}}
+   </center>
+   <div class="container">
+      <table class="table table-condensed">
+         <thead>
+            <tr>
+               <th ng-repeat="data in tableheading">{{data}}</th>
+            </tr>
+         </thead>
+         <tbody>
+            <tr>
+               <td>John</td>
```

Comment:
sir action button implemented as button will come dynamically from the controller.
Also removed harcoded and will use ng-repeat when i fire api to receive data.